front/src/common/redux/sagas/studentRegisteredSage.js

```diff
@@ -0,0 +1,15 @@
+import { put, call } from 'redux-saga/effects';
+import { studentRegistrationService } from '../services/sudentRegistrationService';
+
+import {REGISTRATION_STUDENT_ERROR, REGISTRATION_STUDENT } from '../actions/types';
+
+export function* studentRegisteredSage(payload) {
+  try {
+    const response = yield call(studentRegistrationService, payload);
```

Comment:
You pass here full action, maybe it is better to take only data from that action and pass it in the method